### ISSUE TITLE
Allow finding moves across multiple root modules

### ISSUE THREAD

**Issue by busser** [busser/tfautomv#52]
Currently `tfautomv` only supports running `terraform plan` in a single [root module](https://developer.hashicorp.com/terraform/language/modules#the-root-module) and finding moves to be done in that module. We would like to extend `tfautomv` to be able to run `terraform plan` in any number of root modules and to find moves within _and across_ these modules.

Terraform's `moved` blocks don't work across root modules but we can achieve the desired result with the `terraform state mv`, `terraform state pull`, and `terraform state push` commands.

This issue will keep track of progress made on this feature.

---

Must haves:

- [ ] Allow `tfautomv` to take a list of root module paths as arguments
- [ ] Update the `terraform.Move` value object to include the source and target root modules.
- [ ] Update the plan step to allow running `terraform plan` in multiple directories
- [ ] Update the analysis step to compare resources from any number of plans
- [ ] Update the output step to generate the necessary commands

Nice to haves:

- [ ] Run all `terraform plan` commands in parallel
- [ ] Generate a `moved` block for a move within a root module, and `terraform` commands for a move across root modules

**Comment by busser** [comment]
Achieved in v0.6 :)
